### PR TITLE
Set json logs and output file status

### PR DESCRIPTION
**Pull request by MancunianSam**
- Set logs to output in json format.
- Log the file status so we can query this in Grafana

### CODE REVIEW

File: src/main/scala/uk/gov/nationalarchives/tdr/api/service/FFIDMetadataService.scala [hunk 1]
```diff
@@ -35,6 +39,11 @@ class FFIDMetadataService(ffidMetadataRepository: FFIDMetadataRepository, matche
 
     val fileStatusRows = generateFileStatusRows(ffidMetadata)
 
+    logger.info("File check {} for fileId {} completed with status {}",
```
Comment (suzannehamilton) [inline src/main/scala/uk/gov/nationalarchives/tdr/api/service/FFIDMetadataService.scala:42]: How does this appear in the logs? It looks like there's some string interpolation going on, but do the values below end up in separate JSON fields as well for querying?

File: src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala [hunk 2]
```diff
@@ -46,6 +51,12 @@ class FileMetadataService(fileMetadataRepository: FileMetadataRepository,
             case None => throw new IllegalStateException(s"Cannot find client side checksum for file ${addFileMetadataInput.fileId}")
           }
           fileStatusRow: FilestatusRow = FilestatusRow(uuidSource.uuid, addFileMetadataInput.fileId, Checksum, fileStatus, timestamp)
+          _ <- Future {
+            logger.info("File check {} for fileId {} completed with status {}",
```
Comment (suzannehamilton) [inline src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala:55]: If the structure of these three messages needs to be kept consistent for querying, then could you extract a shared method to do it?

File: src/main/scala/uk/gov/nationalarchives/tdr/api/service/FFIDMetadataService.scala (hunk 1 above)
Comment (MancunianSam) [reply src/main/scala/uk/gov/nationalarchives/tdr/api/service/FFIDMetadataService.scala:42]: Yep, you get the string as a message but you also get the fields included separately. This is an example that I got from running the tests.
```json
{
   "@timestamp":"2021-05-17T11:04:55.433+01:00",
   "@version":"1",
   "message":"File check FFID for fileId 07a3a4bd-0281-4a6d-a4c1-8fa3239e1313 completed with status Success",
   "logger_name":"FFIDMetadataService",
   "thread_name":"consignmentApi-akka.actor.default-dispatcher-6",
   "level":"INFO",
   "level_value":20000,
   "fileCheck":"FFID",
   "fileId":"07a3a4bd-0281-4a6d-a4c1-8fa3239e1313",
   "fileCheckStatus":"Success",
   "application":"consignment-api"
}
```

File: src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala (hunk 2 above)
Comment (MancunianSam) [reply src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala]: Yeah, makes sense. I've created a LoggingUtils class and moved it in there.

File: src/main/scala/uk/gov/nationalarchives/tdr/api/service/FFIDMetadataService.scala (hunk 1 above)
Comment (suzannehamilton) [reply src/main/scala/uk/gov/nationalarchives/tdr/api/service/FFIDMetadataService.scala]: That's really nice!

**Review by suzannehamilton — APPROVED** [review]

**Pull request merged**